.github/workflows/run-examples.yml

```diff
@@ -0,0 +1,46 @@
+name: examples
+
+on:
+  push:
+    # branches: [ master ]
+  pull_request:
+    # branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    name: Run examples
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -r examples/water_boiler/requirements.txt
+        # The pyqt5 package may not be needed for non-interactive runs (such as
+        # in this CI), but may be needed as a backend for Matplotlib to run
+        # interactively.
+        # pip install pyqt5
+
+    - name: Run tests with pytest
```

Comment:
Should probably be "Run examples" or something instead.

Reply:
Good catch! Was a copy-paste leftover.